src/express-api/query.ts

```diff
@@ -0,0 +1,70 @@
+import { newLogger } from "@subsocial/utils"
+import { pg } from '../connections/connect-postgres'
+import { Activity } from '@subsocial/types'
+
+const log = newLogger('ExpressOffchainQuery')
+
+type ActivitiesParams = {
+  account: string,
+  offset: number,
+  limit: number
+}
+
+type Table = 'news_feed' | 'notifications'
+
+const createActivityQuery = (table: Table) => 
+  `SELECT DISTINCT * 
+    FROM df.activities
+    WHERE id IN (
+      SELECT activity_id
+      FROM df.${table}
+      WHERE account = $1 AND aggregated = true
+    )
+    ORDER BY date DESC
+    OFFSET $2
+    LIMIT $3`
+
+const createCountQuery = (table: Table) => 
+  `SELECT COUNT(*)
+    FROM df.activities
+    WHERE id IN (
+      SELECT activity_id
+      FROM df.${table}
+      WHERE account = $1
+    )`
+
+const getQuery = async (
+    query: string,
+    params: any[],
+    errorMsg: string
+  ): Promise<any> => {
+  try {
+    const data = await pg.query(query, params)
+    return data.rows
+  } catch (err) {
+    log.error(errorMsg, err.stack);
+    throw err.stack
```

Comment:
I am not sure that it is a good idea to throw error here. It can stop the whole program?